portal-web/test/functional/com/liferay/portalweb/macros/upgrades/ValidateDBStoreUpgrade.macro

```diff
@@ -0,0 +1,101 @@
+definition {
+
+	macro viewDBStoreUpgrade {
+		task ("Assert no errors are thrown during upgrade") {
+			AssertConsoleTextNotPresent(value1 = "No errors thrown during upgrade");
+		}
+
+		task ("Assert document library is not populated in file system") {
+			var liferayHome = PropsUtil.get("liferay.home.dir.name");
+
+			var dlFolderExists = FileUtil.exists("${liferayHome}/data/document_library");
+
+			if (${dlFolderExists} == "true") {
+				var fileList = FileUtil.listFiles("${liferayHome}/data/document_library");
+
+				var listSize = ListUtil.size(${fileList});
+
+				var dlCount = MathUtil.isGreaterThan(${listSize}, 1);
```

Comment:
I'm checking if there's more than 1 files in document_library because a README.txt gets generated.